### PR TITLE
1433 PV Translations Indonesia and Arabic

### PR DESCRIPTION
**Pull request by Louis-Matsika**
Resolves JIRA [[1344](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1433)]

Overall changes
======
Added translations for portrait video to service configs for Indonesia and Arabic

Newly added translations are passsed into the portrait video title

If no translation is available then default to ‘Watch Moments’

Testing
======
1. vist: `http://localhost.bbc.com:7080/arabic/articles/c35r1w9v8rmo?renderer_env=test`, the title of the PV should say '_لحظات للمشاهدة_'
2. vist: `http://localhost.bbc.com:7080/indonesia/articles/c7x67emlzrxo?renderer_env=test`, the title of the PV should say '_Saksikan juga_'
3. vist a service that should not be affected by this work e.g: `http://localhost.bbc.com:7080/persian/articles/c149pnldynxo?renderer_env=live`, the title of the PV should say '_Watch Moments_'

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/lib/config/services/ws.ts
Comment (Louis-Matsika) [inline src/app/lib/config/services/ws.ts:1]: Screenshot 2024-12-03 at 4 39 36 PM

With implementing the new PV translations for Arabic and Indonesia like this I would have to go to each service in the configs and also give them a default ‘watchMoments’ title for PV.

Is this way to go? or is there another approach I can take that will only require changes to the relevant services?

Comment (karinathomasbbc) [reply src/app/lib/config/services/ws.ts:1]: I think this is the right approach, however I did notice that `watchMoments` is mandatory, when it should probably be optional (as not all services will need this translation). If we make it optional, then this should stop these TS errors 🤞

File: src/app/components/MediaLoader/index.tsx [hunk 1]
```diff
@@ -238,7 +238,11 @@ const MediaLoader = ({ blocks, className, embedded }: Props) => {
         )
       }
       {showPortraitTitle && (
-        <strong css={styles.titlePortrait}>Watch Moments</strong>
+        <strong css={styles.titlePortrait}>
+          {translations.media.watchMoments
+            ? translations.media.watchMoments
+            : 'Watch Moments'}
```
Comment (karinathomasbbc) [inline src/app/components/MediaLoader/index.tsx]: Could this be simplified: 
```suggestion
          {translations.media.watchMoments ||  'Watch Moments' }
```

File: src/app/lib/config/services/ws.ts
Comment (Louis-Matsika) [reply src/app/lib/config/services/ws.ts:1]: Yeahh making it optional makes more sense, pushing changes now

File: src/app/lib/config/services/ws.ts [hunk 2]
```diff
@@ -200,6 +200,7 @@ export const service: DefaultServiceConfig = {
         video: 'Video',
         listen: 'Listen',
         watch: 'Watch',
+        watchMoments: 'Watch Moments',
```
Comment (karinathomasbbc) [inline src/app/lib/config/services/ws.ts]: Do we need to include this translation?

Comment (Louis-Matsika) [reply src/app/lib/config/services/ws.ts]: I re-added this as I thought it was the cause of some errors, but this can be removed. it's not needed

Comment (karinathomasbbc) [reply src/app/lib/config/services/ws.ts]: Cool, thanks for sorting that.

**Review by karinathomasbbc — APPROVED** [review]

**Review by Isabella-Mitchell — APPROVED** [review]
Nice!

We can also see the changes on Storybook: [Arabic](https://1433-pv-translations--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&globals=service.service%3Aarabic&id=components-medialoader--portrait&viewMode=story), [Indonesian](https://1433-pv-translations--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&globals=service.service%3Aindonesia&id=components-medialoader--portrait&viewMode=story), [Another service](https://1433-pv-translations--5d28eb3fe163f6002046d6fa.chromatic.com/iframe.html?args=&globals=service.service:mundo&id=components-medialoader--portrait)

**Pull request merged**